interface/orders/order_manifest.php

```diff
@@ -178,162 +178,161 @@ function generate_order_summary($orderid)
+                "FROM procedure_answers AS a " .
+                "LEFT JOIN procedure_questions AS q ON " .
+                "q.lab_id = ? " .
+                "AND q.procedure_code = ? AND " .
+                "q.question_code = a.question_code " .
+                "WHERE " .
+                "a.procedure_order_id = ? AND " .
+                "a.procedure_order_seq = ? " .
+                "ORDER BY q.seq, a.answer_seq",
+                array($lab_id, $procedure_code, $orderid, $order_seq)
+            );
+
+            $notes = '';
+            while ($qrow = sqlFetchArray($qres)) {
+                // Formatting of these answer values may be lab-specific and we'll figure
+                // out how to deal with that as more labs are supported.
+                $answer = trim($qrow['answer']);
+                $fldtype = $qrow['fldtype'];
+                if ($fldtype == 'G') {
+                    $weeks = intval($answer / 7);
+                    $days = $answer % 7;
+                    $answer = $weeks . 'wks ' . $days . 'days';
+                }
+
+                if ($notes) {
+                    $notes .= '<br />';
+                }
+
+                $notes .= text($qrow['question_text'] . ': ' . $answer);
             }
 
-            if ($notes) {
-                $notes .= '<br />';
+            if ($notes === '') {
+                $notes = '&nbsp;';
             }
 
-            $notes .= text($qrow['question_text'] . ': ' . $answer);
-        }
-
-        if ($notes === '') {
-            $notes = '&nbsp;';
-        }
+            ++$encount;
+            $bgcolor = "#" . (($encount & 1) ? "ddddff" : "ffdddd");
+            echo " <tr class='detail'style='background-color: $bgcolor;'>\n";
+            echo "  <td><input type='checkbox' name='form_omit[" . attr($order_seq) . "]' value='1'";
+            if (!empty($row['do_not_send'])) {
+                echo " checked";
+            }
 
-        ++$encount;
-        $bgcolor = "#" . (($encount & 1) ? "ddddff" : "ffdddd");
-        echo " <tr class='detail' bgcolor='$bgcolor'>\n";
-        echo "  <td><input type='checkbox' name='form_omit[" . attr($order_seq) . "]' value='1'";
-        if (!empty($row['do_not_send'])) {
-            echo " checked";
+            echo " /></td>\n";
+            echo "  <td>" . myCellText("$procedure_code") . "</td>\n";
+            echo "  <td>" . myCellText("$procedure_name") . "</td>\n";
+            echo "  <td>" . myCellText("$diagnoses") . "</td>\n";
+            echo "  <td>$notes</td>\n";
+            echo " </tr>\n";
         }
-
-        echo " /></td>\n";
-        echo "  <td>" . myCellText("$procedure_code") . "</td>\n";
-        echo "  <td>" . myCellText("$procedure_name") . "</td>\n";
-        echo "  <td>" . myCellText("$diagnoses") . "</td>\n";
-        echo "  <td>$notes</td>\n";
-        echo " </tr>\n";
-    }
-    ?>
-
-</table>
+        ?>
+    </table>
 </div>
```

Comment:
In the `@media print` style on line 367 in the body style, add `font-size: 12pt` to the style to allow print compatibility.